arkouda/strings.py

```diff
@@ -2426,3 +2513,8 @@ def unregister_strings_by_name(user_defined_name: str) -> None:
         register, unregister, attach, is_registered
         """
         unregister_pdarray_by_name(user_defined_name)
+        registry = list_registry()
+        if f"{user_defined_name}_bytes" in registry:
+            unregister_pdarray_by_name(user_defined_name + "_bytes")
+        if f"{user_defined_name}_offsets" in registry:
+            unregister_pdarray_by_name(user_defined_name + "_offsets")
```

Comment:
Same argument to be made here as `util/attach` from my last review. We're double defining the name for bytes/offsets in 2 places and 2 different ways